package.json

```diff
@@ -99,7 +99,7 @@
     "typescript": "^4.0.2",
     "uglify-js": "^3.5.8",
     "underscore-template-loader": "^1.0.0",
-    "webdriver-manager": "^12.1.7",
+    "webdriver-manager": "^12.1.8",
```

Comment:
Uhm, you don't need to do this, you just need to run `yarn upgrade` that will generate new yarn.lock file. When `^` is in front of the version patch and minor updates are upgraded automatically, only for major versions the number needs to be changed manually.

I will be soon merging #11754 (tests are flaky there), where `webdriver-manager` is upgraded so no need for any PR from your side.

Reply:
Wouldn't it be better to update this line anyway, since we _do not_ want 12.1.7 to be a valid version anymore? I think it still makes sense to say "minimum version should be 12.1.8, not 12.1.7", right?

Reply:
I guess you can do that.